test/Microsoft.Azure.Devices.Edge.Test.Common/IotHub.cs

```diff
@@ -82,13 +82,14 @@ public async Task<Device> CreateDeviceIdentityAsync(Device device, CancellationT
             return await this.RegistryManager.AddDeviceAsync(device, token);
         }
 
-        public Task<Device> CreateEdgeDeviceIdentityAsync(string deviceId, CancellationToken token)
+        public Task<Device> CreateEdgeDeviceIdentityAsync(string deviceId, AuthenticationType authType, X509Thumbprint x509Thumbprint, CancellationToken token)
```

Comment:
Could you introduce safe-guard in case of either `authType` or `x509Thumbprint` is a null please?

Reply:
But it's desirable that they have the option to be null :)